mirai-core-api/src/commonMain/kotlin/data/RequestEventData.kt

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2019-2021 Mamoe Technologies and contributors.
+ *
+ *  此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ *  Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
+ *
+ *  https://github.com/mamoe/mirai/blob/master/LICENSE
+ */
+
+@file:Suppress("unused")
+
+package net.mamoe.mirai.data
+
+import kotlinx.serialization.Serializable
+import net.mamoe.kjbb.JvmBlockingBridge
+import net.mamoe.mirai.Bot
+import net.mamoe.mirai.Mirai
+import net.mamoe.mirai.event.events.BotInvitedJoinGroupRequestEvent
+import net.mamoe.mirai.event.events.MemberJoinRequestEvent
+import net.mamoe.mirai.event.events.NewFriendRequestEvent
+import net.mamoe.mirai.utils.MiraiExperimentalApi
+
+/**
+ * 用于中转各类 `RequestEvent` 的数据体,
+ * 未来可能会添加新字段
+ *
+ * @since 2.7-M2
```

Comment:
```suggestion
 * @since 2.7
```

公开 API 通常不需要告知自哪个 milestone 支持